executor/licenses/repo.txt

```diff
@@ -1,42 +1,84 @@
 seldonio/seldon-core
-GoogleCloudPlatform/gcloud-golang
+googleapis/google-cloud-go
+census-ecosystem/opencensus-go-exporter-ocagent
 census-ecosystem/opencensus-go-exporter-prometheus
 Azure/azure-sdk-for-go
+Azure/go-ansiterm
 Azure/go-autorest
 BurntSushi/toml
+BurntSushi/xgb
 NYTimes/gziphandler
 PuerkitoBio/purell
 PuerkitoBio/urlesc
+Shopify/sarama
```

Comment:
Why is `sarama` also included? Wasn't that part of the Kafka PR?

Reply:
Interesting. Due to Zipkin.

https://github.com/openzipkin/zipkin-go/blob/ef1b34462145c161724adcc13be368269c29019e/go.mod#L4

Reply:
No it wasn't part of the kafka PR yet.